### ISSUE TITLE
FileNotFoundException: ..\System.Runtime.InteropServices.RuntimeInformation.dll does not exist

### ISSUE THREAD

**Issue by dromerolovo** [juicycleff/flutter-unity-view-widget#555]
Hey guys, I encounter this error everytime I use "Export Android" from the flutter tab in Unity. 

Im using Unity 2020.3.19f1, but also happens with Unity 2020.2.2f1. I've done all the steps of the package documentation. 
OS: Windows 11

The console output is the following

FileNotFoundException: C:\Users\username\OneDrive\Documents\Testing\unity-flutter-project\unity_flutter_project\unity\flutter_unity\Builds/flutter_unity.apk\unityLibrary\src\main\Il2CppOutputProject\IL2CPP\build\deploy\netcoreapp3.1\System.Runtime.InteropServices.RuntimeInformation.dll does not exist
System.IO.File.Copy (System.String sourceFileName, System.String destFileName, System.Boolean overwrite) (at <695d1cc93cca45069c528c15c9fdd749>:0)
Build.Copy (System.String source, System.String destinationPath) (at Assets/FlutterUnityIntegration/Editor/Build.cs:168)
Build.DoBuildAndroid (System.String buildPath, System.Boolean isPlugin) (at Assets/FlutterUnityIntegration/Editor/Build.cs:63)
Build.DoBuildAndroidLibrary () (at Assets/FlutterUnityIntegration/Editor/Build.cs:26).

When I try to export directly in Build Setting, there is no problems.

Appreciate the help.

**Comment by cc-setwo** [comment]
Hello,

Looks like you have modified the Build.cs, because I can't find the line, which causes the problem.
The first and silly proposal: do you have the .net framework installed on your pc?
The second proposal: all in all you will need System.Runtime.InteropServices.RuntimeInformation.dll, while building android. You may export it in Build Setting, but make sure you have netcoreapp3.1 libraries under export path   ../unityLibrary/src/main/Il2CppOutputProject/IL2CPP/build/deploy/netcoreapp3.1. If don't you can take mine from zip file attached: https://anonfiles.com/hcYbm0S7x6/netcoreapp3.1_windows_zip

Regards.

**Comment by dromerolovo** [comment]
Vlad, thanks for answering.

Regarding the Build.cs file. I imported the package of the repo flutter-unity-view-widget/FlutterUnityIntegration-v4.1.0-WithDemo.unitypackage. And, then I changed some lines of code following the troubleshooting guide. I dint't do any more changes.

1.1. On line 48, change the following:

-    var options = BuildOptions.AcceptExternalModificationsToPlayer;
+    var options = BuildOptions.AllowDebugging;
+    EditorUserBuildSettings.exportAsGoogleAndroidProject = true;
1.2. On line 115, change the following:

-    var options = BuildOptions.AcceptExternalModificationsToPlayer;
+    var options = BuildOptions.AllowDebugging;

The lines that are throwing the exeption are the following, according to the console output:

```
[MenuItem("Flutter/Export Android %&n", false, 1)]
    public static void DoBuildAndroidLibrary()
    {
        DoBuildAndroid(Path.Combine(apkPath, "unityLibrary"), false);  <======= This one

        // Copy over resources from the launcher module that are used by the library
        Copy(Path.Combine(apkPath + "/launcher/src/main/res"), Path.Combine(androidExportPath, "src/main/res"));
    }
```

And 

```
public static void DoBuildAndroid(String buildPath, bool isPlugin)
    {
        if (Directory.Exists(apkPath))
            Directory.Delete(apkPath, true); <======== This one

        if (Directory.Exists(androidExportPath))
            Directory.Delete(androidExportPath, true);

        EditorUserBuildSettings.androidBuildSystem = AndroidBuildSystem.Gradle;
```

Regarding NET framework, yes i have installed NET framework, 

Regarding the second proposal, when I export the project using Build Settings, the netcoreapp 3.1 libraries are exported fine. 

What could be a workaraound, maybe exporting the project with Build Settings, and then adding or modifying files so that the files are the same as if they had been exported by the other method( Flutter tab , android export) , IDK. 

I'll keep you guys updated if I fix this or if I make some progress.
